storageClient/storageClient.go

```diff
@@ -58,6 +58,25 @@ func (c *StorageClient) CreateGroup(ctx context.Context, params CreateGroupReque
 	return result, err
 }
 
+// getGroupByGroupNameAndAccountID fetches a group using the specified parameters,
+// returning the group and error (if any).
+func (c *StorageClient) getGroupByGroupNameAndAccountID(ctx context.Context, groupName string, accountID string) (*Group, error) {
```

Comment:
https://github.com/tozny/e3db-clients-go/blob/master/identityClient/identityClient.go#L65
this should be named at a level one abstraction layer higher then the http method and params, e.g. DescribeGroup and collect the group name and account id individual params into a DescribeGroupRequest struct